algorithms/arrays/flatten.py

```diff
@@ -26,6 +26,6 @@ def flatten_iter(iterable):
     """
     for element in iterable:
         if isinstance(element, Iterable):
-            yield from flatten_iter(element)
+            yield from flatten_iter(element)    
```

Comment:
please remove the trailing spaces at the end.
There should be an automatic setting for this.